commit 95c553c46770f89f09c4408cbdf61ed76b4a8766
Author: Sergey <sergemp@mail.ru>
Date:   Thu Oct 20 14:28:39 2011 +0300

    Ticket #2635: mc adds spaces an the beginning of all files/dirs at the ftp server
    
    For FTP servers, that do not send ".." in directory listing, mc adds one space to the name of every file and directory. Attached patch should fix this problem.
    
    Steps to reproduce:
    cd /#ftp:ftp.symantec.com
    
    Expected result:
    Correct list of files/directories
    
    Actual result:
    File/directory names with leading spaces
    
    Signed-off-by: Slava Zanko <slavazanko@gmail.com>

diff --git a/lib/vfs/parse_ls_vga.c b/lib/vfs/parse_ls_vga.c
index 8091c14..5424ee9 100644
--- a/lib/vfs/parse_ls_vga.c
+++ b/lib/vfs/parse_ls_vga.c
@@ -657,7 +657,7 @@ vfs_split_text (char *p)
 void
 vfs_parse_ls_lga_init (void)
 {
-    vfs_parce_ls_final_num_spaces = 0;
+    vfs_parce_ls_final_num_spaces = 1;
 }
 
 /* --------------------------------------------------------------------------------------------- */
